File: InMotionGIT.Common/Services/Contracts/ConnectionString.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class ConnectionString

        <DataMember()>
        Public Property Name As String

        <DataMember()>
        Public Property ConnectionString As String

        <DataMember()>
        Property ProviderName As String

        <DataMember()>
        Public Property UserName As String

        <DataMember()>
        Public Property Password As String

        <DataMember()>
        Public Property Owners As String

        <DataMember()>
        Public Property ServiceName As String

        <DataMember()>
        Public Property DatabaseName As String

        <DataMember()>
        Public Property SourceType As Enumerations.EnumSourceType

    End Class

End Namespace

File: Common/InMotionGIT.Common/Services/Contracts/ConnectionString.cs
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Services.Contracts
{
    /// <summary>
    /// 
    /// </summary>
    [DataContract()]
    public class ConnectionStrings
    {

        [DataMember()]
        public string Name { get; set; }

        /// <summary>
        /// 
        /// </summary>
        [DataMember()]
        public string ConnectionString { get; set; }

        [DataMember()]
        public string ProviderName { get; set; }

        [DataMember()]
        public string UserName { get; set; }

        [DataMember()]
        public string Password { get; set; }

        [DataMember()]
        public string Owners { get; set; }

        [DataMember()]
        public string ServiceName { get; set; }

        [DataMember()]
        public string DatabaseName { get; set; }

        [DataMember()]
        public Enumerations.EnumSourceType SourceType { get; set; }

    }

}